packages/shared_preferences/shared_preferences_tool/pubspec.yaml

```diff
@@ -8,11 +8,11 @@ environment:
   sdk: '>=3.4.0 <4.0.0'
 
 dependencies:
-  devtools_app_shared: ^0.1.1
-  devtools_extensions: ^0.1.1
+  devtools_app_shared: ^0.2.2
```

Comment:
@kenzieschmoll  I couldn't use `0.2.3` here, it uses `web 1.0.0`.

```
Resolving dependencies... 
Because no versions of devtools_extensions match >0.2.2 <0.3.0 and devtools_extensions 0.2.2 depends on web ^0.5.0, devtools_extensions ^0.2.2 requires web ^0.5.0.
And because devtools_app_shared >=0.2.3 depends on web ^1.0.0, devtools_extensions ^0.2.2 is incompatible with devtools_app_shared >=0.2.3.
So, because shared_preferences_tool depends on both devtools_app_shared ^0.2.3 and devtools_extensions ^0.2.2, version solving failed.
```